sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py

```diff
@@ -75,11 +74,11 @@
         'azure.monitor',
         'azure.monitor.opentelemetry'
     ]),
-    python_requires=">=3.5.0",
+    python_requires=">=3.6.0",
     install_requires=[
         "azure-core<2.0.0,>=1.6.0",
         "msrest>=0.6.10",
-        "opentelemetry-api == 1.0.0rc1",
-        "opentelemetry-sdk == 1.0.0rc1"
+        "opentelemetry-api == 1.0.0",
+        "opentelemetry-sdk == 1.0.0"
```

Comment:
let's not pin this anymore since it's GA and we expect OT to follow semver
```suggestion
        "opentelemetry-api<2.0.0,>=1.0.0,
        "opentelemetry-sdk<2.0.0,>=1.0.0"
```